.ci/prepare-settings.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+date=$(date +"%m%d%y")
+
+# Copying release-settings.xml
+mkdir -p .ci-temp
+cp ./.ci/release-settings.xml .ci-temp/
+
+# Putting values of environment variables in .ci-temp/release-settings.xml
+replace() {
+  sed -i "s/$1/$2/g" .ci-temp/release-settings.xml
+}
+
+  replace SONATYPE_USER romanivanov
+  replace SONATYPE_PWD password
```

Comment:
Make method indentation be 0.

Please remove my names and other bummy values.
If variable is not resolved it stays as is in file